lib/vagrant-service-manager/command.rb

```diff
@@ -86,8 +87,20 @@ def check_if_a_service_is_running?(service)
 
       def print_all_provider_info
         @env.ui.info I18n.t('servicemanager.commands.env.nil')
-        self.execute_docker_info
-        self.execute_openshift_info
+
+        running_services = []
+        SUPPORTED_SERVICES.each do |service|
```

Comment:
You may want to change these `each` blocks as:

```
        SUPPORTED_SERVICES.each do |service|
          status = if check_if_a_service_is_running?(service)
                     running_services << service
                     'running'
                   else
                     'stopped'
                   end
          @env.ui.info("#{service.capitalize} - #{status}")
        end
        running_services.each do |service|
          @env.ui.info("\n#{service.capitalize} env:")
          public_send("execute_#{service}_info")
        end
``` 
* Capitalization of service name
* One time usage of `@env.ui.info` 

Rest LGTM.